celements-spring-mvc/src/main/java/com/celements/spring/mvc/CelMvcConfig.java

```diff
@@ -25,18 +16,4 @@
 })
 public class CelMvcConfig implements WebMvcConfigurer {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(CelMvcConfig.class);
-
-  private final List<CelMvcInterceptor> interceptors;
-
-  @Inject
-  public CelMvcConfig(List<CelMvcInterceptor> interceptors) {
-    this.interceptors = ImmutableList.copyOf(interceptors);
-  }
-
-  @Override
-  public void addInterceptors(InterceptorRegistry registry) {
-    LOGGER.trace("addInterceptors: size {}", interceptors.size());
-    interceptors.forEach(registry::addInterceptor);
-  }
```

Comment:
I'd keep the CelMvcInterceptor as a framework feature to easily add custom interceptors in the future.